src/vxingest/netcdf_to_cb/run_ingest_threads.py

```diff
@@ -193,7 +196,7 @@ def runit(self, args, log_queue: Queue, log_configurer: Callable[[Queue], None])
             self.load_spec["fmask"] = self.fmask
             self.load_spec["input_data_path"] = self.path
             # stash the load_job in the load_spec
-            self.load_spec["load_job_doc"] = self.build_load_job_doc("madis")
+            self.load_spec["load_job_doc"] = self.build_load_job_doc("tropoe")
```

Comment:
Does hardcoding this to `tropoe` break the ingest for MADIS (/METAR?) data?

Reply:
good catch